src/genie/libs/parser/iosxe/tests/test_show_cts.py

```diff
@@ -51,6 +52,153 @@ def test_show_cts_pacs_empty(self):
+            },
+            3: {
+                "src_grp_id": 199,
+                "src_grp_name": "FAILED_AUTH",
+                "unknown_group": "Unknown",
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            4: {
+                "src_grp_id": 22,
+                "src_grp_name": "Untrusted",
+                "dst_group_id": 22,
+                "dst_group_name": "Untrusted",
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            5: {
+                "src_grp_id": 100,
+                "src_grp_name": "Tech",
+                "dst_group_id": 6,
+                "dst_group_name": "Trusted",
+                "policy_groups": {
+                    1: {
+                        "policy_group": "NFS-08"
+                    },
+                    2: {
+                        "policy_group": "ACCESS-01"
+                    }
+                },
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            6: {
+                "src_grp_id": 101,
+                "src_grp_name": "Mars",
+                "dst_group_id": 5,
+                "dst_group_name": "Trusted",
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            7: {
+                "src_grp_id": 111,
+                "src_grp_name": "Technicolor",
+                "dst_group_id": 9,
+                "dst_group_name": "Printers",
+                "policy_groups": {
+                    1: {
+                        "policy_group": "PRINTING-10"
+                    },
+                    2: {
+                        "policy_group": "GENERIC-14"
+                    }
+                },
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            8: {
+                "src_grp_id": 102,
+                "src_grp_name": "Moon",
+                "dst_group_id": 6,
+                "dst_group_name": "Printers",
+                "policy_groups": {
+                    1: {
+                        "policy_group": "ICMP-01"
+                    },
+                    2: {
+                        "policy_group": "PRINTING-10"
+                    }
+                },
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            9: {
+                "src_grp_id": 199,
+                "src_grp_name": "FAILED_AUTH",
+                "dst_group_id": 9999,
+                "dst_group_name": "QUARANTINE_REDIRECT_PORTAL",
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            "monitor_dynamic": False,
+            "monitor_configured": False
+        }
+    }
+
+    golden_output1 = {'execute.return_value': '''
+    IPv4 Role-based permissions default:
```

Comment:
Can you fix the spacing on this, every line would have 4 spaces with this format.